src/app/ReadHandler.cpp

```diff
@@ -128,16 +173,35 @@ CHIP_ERROR ReadHandler::OnStatusReport(Messaging::ExchangeContext * apExchangeCo
 
 CHIP_ERROR ReadHandler::SendReportData(System::PacketBufferHandle && aPayload)
 {
+    CHIP_ERROR err = CHIP_NO_ERROR;
```

Comment:
used in only one place, please declare when used instead of  at the top of the function.